### PR TITLE
#2183 Add container table loader and typography component unit tests

### PR DESCRIPTION
**Pull request by DavidMi-1**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2183)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Mui/Typography/Typography.js [hunk 1]
```diff
@@ -18,9 +18,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 
-const MuiTypography = ({ variant, children, ...rest }) => {
+const MuiTypography = ({ variant, children, testid, ...rest }) => {
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Typography/Typography.js]: Please assign a default value to the testid, and it should be camel case like `testId` not `testid` :)

File: ohara-manager/client/src/components/common/Mui/Typography/__tests__/Typography.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    jest.clearAllMocks();
+  });
+
+  it('renders typography', async () => {
+    render(<Typography {...setup()} />);
+  });
+
+  it('renders typography with assigned style class', async () => {
+    const props = setup({ variant: 'h3', testid: 'mui-style' });
+    const { getByTestId } = await waitForElement(() =>
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Typography/__tests__/Typography.test.js]: Is the await necessary here?

Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Typography/__tests__/Typography.test.js]: It does not seem to need. I'll remove them. :)

File: ohara-manager/client/src/components/common/Mui/Typography/Typography.js (hunk 1 above)
Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Typography/Typography.js]: Oop, I forgot the default value after created new PR, lol

File: ohara-manager/client/src/components/common/Mui/Typography/Typography.js [hunk 3]
```diff
@@ -18,9 +18,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 
-const MuiTypography = ({ variant, children, ...rest }) => {
+const MuiTypography = ({
+  variant,
+  children,
+  testId = 'typo-graphy',
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Typography/Typography.js]: This is actually a real English word :)
```suggestion
  testId = 'typography',
```

Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Typography/Typography.js]: orz, got it, lol

**Comment by konekoya** [comment]
Thanks, @DavidMi-1 :)

**Pull request merged**